initrd/bin/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+. /etc/luks-functions
+
+enable_usb
+enable_usb_storage
+prepare_thumb_drive /dev/sda 25 "test"
```

Comment:
Is this going to appear in menus anywhere or just something we suggest to do from the recovery shell for now?

Reply:
@JonathonHall-Purism will be used under non-default oem-factory-reset/Re-Ownership to create GPG backup thumb drive of in-memory created gpg key material under LUKS encrypted partition, where public partition used to store public key. And then Thumb drive is useable in case USB Security dongle is lost if that option was selected on Re-Ownership, until USB Security dongle replacement is received. And then another PR will enable USB thumb drive subkeys restoration on USB Security dongle received replacement. And finally, OEM Factory Reset will permit to not depend on a USB Security dongle altogether altogether. And then another PR will enable authentication in case USB thumb drive backup of GPG key material exist, which will be useable to test detach sign/verify, which will permit authentication prior of recovery shell access, flashing and USB boot.